src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -79,8 +79,10 @@
         "symfony/mailer": "<4.4",
         "symfony/messenger": "<4.4",
         "symfony/mime": "<4.4",
+        "symfony/notifier": ">=5.1",
```

Comment:
why this one?

Reply:
We register services for the Notifier component in `FrameworkExtension::registerNotifierConfiguration()`.

Reply:
I'm reverting the conflicts here, they make composer unable to resolve deps.

Reply:
yeah, that's because dev-master branches that conflict rule (see https://github.com/composer/composer/issues/8323 where I suggest we should change it in composer 2)

Reply:
So we should re-revert this before the release of Symfony 5.

Reply:
I think we don't need to. IF there is some conflict to add, we will add it to the components 5.1